LocalSettings.php

```diff
@@ -5013,6 +5013,14 @@
 				'upload_by_url' => true,
 			),
 		),
+		'+madgendersciencewiki' => array(
+			'scholar' => array(
+				'editprotected' => true,
+			),
+			'*' => array(
```

Comment:
The edit permission should be revoked using wgEditingMatrix. Basically add an entry for your wiki and only set anon to true.